Azure-Samples/azure-iot-samples-csharp
# added Documentation button to pnp device sample readme and associated…

… png files

## Purpose

* 
Add a Documentation button to the device pnp sample readme per format adopted for other pnp samples used in device developer docs. 

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
You can check the readme listed under Files for a button at the top labeled "Documentation" that resolves here: https://docs.microsoft.com/azure/iot-develop/ .
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
